src/helpers/current-page-helper.php

```diff
@@ -210,4 +210,26 @@ public function is_error_page() {
 
 		return $wp_query->is_404();
 	}
+
+	/**
+	 * Determine whether this page is an taxonomy archive page for multiple terms (url: /term-1,term2/).
+	 *
+	 * @return bool Whether or not the current page is an archive page for multiple terms.
+	 */
+	public function is_multiple_terms_page() {
+		$wp_query = $this->wp_query_wrapper->get_main_query();
+
+		if ( ! $this->is_term_archive() ) {
+			return false;
+		}
+
+		$term          = $wp_query->get_queried_object();
+		$queried_terms = $wp_query->tax_query->queried_terms;
+
+		if ( empty( $queried_terms[ $term->taxonomy ]['terms'] ) ) {
```

Comment:
```suggestion
		if ( \empty( $queried_terms[ $term->taxonomy ]['terms'] ) ) {
```

Reply:
Empty is not a function but a reserved keyword, therefore it cannot be escaped.